core/src/trezor/ui/layouts/tt_v2/reset.py

```diff
@@ -20,15 +20,19 @@
 def _split_share_into_pages(share_words: Sequence[str], per_page: int = 4) -> list[str]:
     pages: list[str] = []
     current = ""
+    fill = 2
 
     for i, word in enumerate(share_words):
         if i % per_page == 0:
             if i != 0:
                 pages.append(current)
             current = ""
+
+            lastnum = i + per_page + 1
```

Comment:
I would welcome a comment on why we do this

Reply:
Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).